Document/0x06b-Basic-Security-Testing.md

```diff
@@ -573,3 +573,28 @@ You can remotely sniff all traffic in real-time on iOS by [creating a Remote Vir
 ```shell
 ip.addr == 192.168.1.1 && http
 ```
+
+### Allow application installation on an non-iPad device
```

Comment:
Can you please adjust the title according to the style guide? 

https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md#title-capitalization

Reply:
BTW: Thanks for the content, that is really useful!

Reply:
It's done, i have changed it to **Allow Application Installation on an Non-Ipad Device**.

I have used the site https://capitalizemytitle.com using `Chicago` style.

The site http://www.titlecapitalization.com/ do not respond:
```
Firefox can't establish a connection to the server at www.titlecapitalization.com.
```